MAINTAINERS_RULES.md

```diff
@@ -0,0 +1,39 @@
+# Maintainers Rules
+
+This document lays out some basic rules and guidelines all maintainers are expected to follow.
+Changes to the [Acceptance Criteria](#hard-acceptance-criteria-for-merging-a-pr) for merging PRs require a ceiling(two-thirds) supermajority from the maintainers.
+Changes to the [Repo Guidelines](#repo-guidelines) require a simple majority.
+
+## Hard Acceptance Criteria for merging a PR:
+
+- 2 LGTMs are required when merging a PR
```

Comment:
both of these LGTMs need to be maintainers? i think we need more maintainers before we can say that.